src/composables/multisigAccountCreate.ts

```diff
@@ -89,63 +88,87 @@ export function useMultisigAccountCreate({ store }: IDefaultComposableOptions) {
         innerTx: true,
       },
     });
-    accountId = gaAccount.publicKey;
-
-    signedAttachTx = await drySdk.signTransaction(attachTX.tx, {
-      innerTx: true,
-      onAccount: gaAccount,
-    });
+    pendingMultisigCreationTxs.value[gaAccount.publicKey]
+      .signedAttachTx = await drySdk.signTransaction(attachTX.tx, {
+        innerTx: true,
+        onAccount: gaAccount,
+      });
+    multisigAccountCreationPhase.value = MULTISIG_CREATION_PHASES.prepared;
+    return gaAccount.publicKey;
   }
 
   /**
    * First step of creating the multisig account
    * Prepare multisig account creation transaction
    */
-  async function prepareVaultCreationRawTx(
-  ) {
+  async function prepareVaultCreationRawTx(payerId: string, accountId: string) {
+    if (
+      !pendingMultisigCreationTxs.value[accountId]
+      && !pendingMultisigCreationTxs.value[accountId].signedAttachTx) {
+      throw Error(`GA Attach Tx not found for account ${accountId}, Prepare attach transaction first`);
+    }
```

Comment:
If you are dropping something in the `if` clause please be consistent with it
```suggestion
    if (
      !pendingMultisigCreationTxs.value[accountId]
      && !pendingMultisigCreationTxs.value[accountId].signedAttachTx
      ) {
      throw Error(`GA Attach Tx not found for account ${accountId}, Prepare attach transaction first`);
    }
```

Reply:
Also here you could shorten the condition to
```suggestion
    if (!pendingMultisigCreationTxs.value[accountId]?.signedAttachTx) {
      throw Error(`GA Attach Tx not found for account ${accountId}, Prepare attach transaction first`);
    }
```

Reply:
Fixed. Thanks